src/components/Award.css

```diff
@@ -0,0 +1,12 @@
+.awards__item {
+    border-bottom: 1px dotted #ccc;
+}
+.awards__item__body {
+    display: none;
+}
+.awards__item__body.active {
+    display: block;
+}
```

Comment:
Not needed in this PR but... you can use `d-block` `d-none` bootstrap utilities instead of introducing new classes. The ratio is always: try to NOT write CSS (or at least try to minimize the custom CSS), use utilities instead. This

1. will lower the development and maintenance effort (less is more)
2. will use a common shared vocabulary between devs (which know bootstrap utilities)
3. will reduce the size of the CSS sent to the browser
4. will keep the styles robust and consistent (think about: avoid introducing custom and inconsistent sizing, colors, etc.)

I suggest to implement a set of commons utilities classes (when they are not covered by the bootstrap ones), ie.
`.opacity-50 { opacity: 0.5 }` trying to reuse them everywhere and avoid writing CSS `at all` (that's the TailwindCSS philosophy ;-)

Aside, there's no need to use BEM notation since the styles are already scoped to the component when using CSS modules (aren't we?)

Reply:
right